src/main/java/com/exasol/adapter/dialects/oracle/OracleProperties.java

```diff
@@ -6,6 +6,7 @@
 public final class OracleProperties {
     public static final String ORACLE_CAST_NUMBER_TO_DECIMAL_PROPERTY = "ORACLE_CAST_NUMBER_TO_DECIMAL_WITH_PRECISION_AND_SCALE";
     public static final String ORACLE_IMPORT_PROPERTY = "IMPORT_FROM_ORA";
+    public static final String IS_LOCAL_PROPERTY = "IS_LOCAL";
```

Comment:
`IS_LOCAL` seems out of place here. This property is for the Exasol VS when projecting a local VS.

Reply:
Good catch, I removed it.